docs/scos/user/intro-to-spryker/contribute-to-documentation/build-the-documentation-site.md

```diff
@@ -28,54 +28,119 @@ We use [Jekyll](https://jekyllrb.com/) to build the Spryker documentation site.
+
+{% endinfo_block %}
+
 
-#### 1. Install command line tools
-To compile native extensions, install the command line tools:
+{% info_block warningBox "MacOS on M1 processor – Open Terminal using Rosetta" %}
 
+On a **MacBook with the M1 processor**, make sure your Terminal is opened with Rosetta 2, by following the instructions here. Rosetta is a translation layer that enables non-native Intel x86 apps, including Homebrew, to run on Apple Silicon Macs. 
+To open your terminal using Rosetta:
+1.	Open a Finder window.
+2.	In Applications, locate Terminal.
+3.	Right click on **Terminal** and select **Get Info**. 
+4.	Make sure _Open using Rosetta_ is checked. 
+  
+{% endinfo_block %}
+
+On either an M1 or an Intel Mac, follow the steps below to install Jekyll. M1-specific instructions appear as notes wherever necessary.  
+
+#### 1. Install Homebrew
+
+Homebrew is a package manager for macOS (because by default, Mac doesn’t have a package manager). You use Homebrew to install Ruby in the next step. Additionally, when you install Homebrew, Xcode command line tools and GCC are also installed automatically. 
+
+To install Homebrew: 
 ```bash
-xcode-select --install
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 ```
+{% info_block infoBox "MacOS on M1 processor" %}
 
-#### 2. Install Ruby
-Check your current Ruby version:
+On an M1 Mac, Homebrew files are installed into the /opt/homebrew folder. Because this folder is not part of the default $PATH, you need to follow the next steps that Homebrew includes at the end of the installation output to add Homebrew to your PATH. In the example below, we've replaced your actual username with:  /Users/_username_/.zprofile
 
 ```bash
-ruby -v
+==> Next steps:
+- Run these three commands in your terminal to add Homebrew to your PATH:
+    echo '# Set PATH, MANPATH, etc., for Homebrew.' >> /Users/_username_/.zprofile
+    echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/_username_/.zprofile
+    eval "$(/opt/homebrew/bin/brew shellenv)"
 ```
-If Ruby is not installed or the version is below 2.4.0, do the following: 
 
- 1. Install Homebrew:
+Alternately, or if you have problems using these commands to update your PATH, you can add Homebrew to your .zprofile file. First, run:
 ```bash
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+nano ~/.zprofile
 ```
-2. Install Ruby:   
+
+If the profile file has not yet been created on your system, this command creates a new one. If the file already exists, this command opens it.
+
+To add Homebrew, paste this text into the .zprofile file open in your terminal:
+```bash
+# Set PATH, MANPATH, etc., for Homebrew.
+eval "$(/opt/homebrew/bin/brew shellenv)"
+```
+Click **control o** to save the updated .zprofile file. Then click **control x** to exit out of the .zprofile file.
```

Comment:
```suggestion
To save the updated .zprofile file, click **control o**. Then, to exit the .zprofile file, click **control x**.
```
It is recommended to state the goal first, and then the action: https://developers.google.com/style/procedures#steps-with-goals